SeaPublicWebsite/Controllers/FeedbackController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SeaPublicWebsite.ExternalServices.EmailSending;
+using SeaPublicWebsite.Models.Feedback;
+
+namespace SeaPublicWebsite.Controllers;
+
+[Route("feedback")]
```

Comment:
Why change this? We're not accepting feedback for all of gov.uk :)

Reply:
Change it from what? I don't understand :)

Reply:
Setting this on the controller makes the route `/feedback` rather than `/energy-efficiency/feedback`